go.sum

```diff
@@ -0,0 +1,30 @@
+github.com/davecgh/go-spew v1.1.0/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
+github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSsI+c5H38=
```

Comment:
why is this package required? it was not present on the `.lock` file

Reply:
I don't know. go.sum and go.mod are created by go module. Maybe a repo used depends on them

Reply:
yes, but I can't find where it comes from

this the output for executing `goda graph -cluster -short ./... | dot -Tpng -o graph.png` on your branch
![graph](https://user-images.githubusercontent.com/1307694/59857564-7d0f6e80-9379-11e9-924a-a8c9577c74fe.png)

this is not a major issue, but I'd keep the deps as clean as possible

Reply:
you are absolutely right. I checked it now using gin-contrib/cors repo.